lib/Suggestions.js

```diff
@@ -17,27 +17,29 @@ function markIt (input, query) {
   }
 }
 
-function filterSuggestions (query, suggestions, length, suggestionsFilter) {
+function filterSuggestions (query, suggestions, length, suggestionsFilter, noSuggestionsMessage) {
   if (!suggestionsFilter) {
     const regex = new RegExp(`(?:^|\\s)${escapeForRegExp(query)}`, 'i')
     suggestionsFilter = (item) => regex.test(item.name)
   }
 
-  return suggestions.filter((item) => suggestionsFilter(item, query)).slice(0, length)
+  let filtered = suggestions.filter((item) => suggestionsFilter(item, query)).slice(0, length)
+
+  return (filtered.length > 0) ? filtered : [{ id: 0, name: noSuggestionsMessage, disabled: true }]
```

Comment:
I think the message should be optional and opt-in (to avoid changing the behaviour for everybody using the component already). With that in kind I think this should be:

```js
const filtered = suggestions.filter((item) => suggestionsFilter(item, query)).slice(0, length)

if (filtered.length === 0 && noSuggestionsMessage) {
  filtered.push({ id: 0, name: noSuggestionsMessage, disabled: true })
}

return filtered
```

Reply:
Yeah, agree, i forgot about that check. Will adjust and commit. 👍